USFMToolsSharp.Renderers.HTML/HTMLConfig.cs

```diff
@@ -17,13 +18,15 @@ public HTMLConfig()
             divClasses = new List<string>();
             separateChapters = false;
             separateVerses = false;
+            blankColumn = false;
             partialHTML = false;
         }
-        public HTMLConfig(List<string> divClasses, bool separateChapters = false, bool partialHTML = false, bool separateVerses=false)
+        public HTMLConfig(List<string> divClasses, bool separateChapters = false, bool partialHTML = false, bool blankColumn = false, bool separateVerses=false)
```

Comment:
You should put the new column at the end of the list.

Reply:
@rbnswartz Done